src/main/kotlin/no/nav/familie/tilbake/kravgrunnlag/batch/HåndterGammelKravgrunnlagTask.kt

```diff
@@ -56,7 +57,19 @@ class HåndterGammelKravgrunnlagTask(
                     "Feiler med $feilMelding"
             )
         }
-        håndterGamleKravgrunnlagService.håndter(hentFagsystemsbehandlingRespons.hentFagsystemsbehandling!!, mottattXml)
+        try {
+            håndterGamleKravgrunnlagService.håndter(hentFagsystemsbehandlingRespons.hentFagsystemsbehandling!!, mottattXml)
+        } catch (e: IntegrasjonException) {
+            if (e.cause?.message?.contains("Kravgrunnlag ikke funnet") == true &&
+                håndterGamleKravgrunnlagService.sjekkArkivForDuplikatKravgrunnlagMedKravstatusAvsluttet(
+                    kravgrunnlagIkkeFunnet = mottattXml
+                )
+            ) {
+                håndterGamleKravgrunnlagService.arkiverKravgrunnlag(mottattXmlId)
```

Comment:
Burde vi logget en liten "warning" her for å kunne finne igjen og forstå hva som har skjedd. For tasken "håndtergammelkravgrunnlag" blir jo ferdigstilt, men den oppretter intet nytt. Fint når man skal debugge å kunne se logginnslag om at den ikke har blitt håndtert som forventet..?

Reply:
Kan være fint, ja 👍